tests/api/test_enrolments.py

```diff
@@ -0,0 +1,93 @@
+from unittest import mock
+
+from fastapi.testclient import TestClient
+
+from app.domain.entities.enrolment_request import EnrolmentRequest
+from app.main import app
+from app.responses import FailureType, ResponseFailure, ResponseSuccess, SuccessType
+from app.security import oauth2_scheme
+from tests.test_data.enrolment_data_provider import EnrolmentDataProvider
+
+test_data = EnrolmentDataProvider()
+client = TestClient(app)
+
+
+async def override_dependency():
+    return "TEST-JWT-TOKEN"
+
+
+app.dependency_overrides[oauth2_scheme] = override_dependency
+
+
+@mock.patch("app.use_cases.get_enrolments.GetEnrolments")
+def test_get_enrolments_success(use_case):
+    message = "The enrolment requests have been processed."
```

Comment:
The enrolment requests have been fetched

Reply:
Fixed